ui/src/adapters/env.ts

```diff
@@ -0,0 +1,46 @@
+import { z } from "zod";
+
+import * as domain from "src/domain";
+import { StrictSchema } from "src/utils/types";
+
+export interface Env {
+  VITE_API_PASSWORD: string;
+  VITE_API_URL: string;
+  VITE_API_USERNAME: string;
+  VITE_ISSUER_DID: string;
+  VITE_ISSUER_LOGO?: string;
+  VITE_ISSUER_NAME?: string;
+}
+
+export const envParser = StrictSchema<Env, domain.Env>()(
```

Comment:
We can't just run this parser on startup and have them available as global vars? Not sure why they need to be handled in a React context if they're static, it only seems to add complexity and prop/param drilling.

Reply:
I didn't realize that the app was rendering an error, I thought it would just fail. This makes sense then.